Foundation/GTMNSThread+Blocks.m

```diff
@@ -53,220 +53,76 @@ + (void)gtm_performBlockInBackground:(void (^)(void))block {
 
 #endif  // NS_BLOCKS_AVAILABLE
 
-enum {
-  kGTMSimpleThreadInitialized = 0,
-  kGTMSimpleThreadStarting,
-  kGTMSimpleThreadRunning,
-  kGTMSimpleThreadCancel,
-  kGTMSimpleThreadFinished,
-};
-
-@implementation GTMSimpleWorkerThread
-
-- (id)init {
-  if ((self = [super init])) {
-    runLock_ =
-        [[NSConditionLock alloc] initWithCondition:kGTMSimpleThreadInitialized];
-  }
-  return self;
+@implementation GTMSimpleWorkerThread {
```

Comment:
In theory this might need to be in the header for continue to support old xcodes/runtimes, but I'd say wait until it is a issue.